### DISCUSSION TITLE
Turbo mode with custom child theme of breeze blank

### DISCUSSION THREAD

**Discussion by sandipansaha** [breezefront/community#54] [1. Help]
Hi Support,

Do we need to do anything additional for the Turbo mode in case of a custom child theme of breeze blank? It feels like the pages are being reloaded.
Is there any way to verify whether the turbo mode is working or not?

Thanks

**Comment by vovayatsyuk** [comment]
Breeze will reload the page if the static content version was changed between visits. In all other cases, it should update the page via ajax.

**Reply by sandipansaha** [reply]
Hello Yatsyuk,
Thank you for your reply. It seems that my pages are reloading. How could I debug this? Could you please help me to understand this so it can use the breeze turbo mode efficiently?
Would be very helpful if you could check the pages - URL
I appreciate your help very much.
Thanks

**Reply by vovayatsyuk** [reply]
I see that all clicks are captured by the `NREUM` script. Could you remove this script from the site?

**Reply by sandipansaha** [reply]
Hello Yatsyuk,

Thank you for your message.

It seems it is coming from the Newrelic services that are part of the Magento cloud infrastructure - https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/monitor/new-relic.html?lang=en

It is recommended and very helpful to manage, monitor, and troubleshoot the applications and infrastructure by collecting, viewing, and analyzing data.

What do you suggest in that case? Is there any way we can use both breeze and newrelic together - maybe in near future?

Thanks
Sandipan

**Reply by vovayatsyuk** [reply]
I don't know if it's the actual reason why the turbo is not working. Can you disable it and check if it helps?

**Reply by sandipansaha** [reply]
Hi Yatsyuk,

It is disabled now, but I still think it is being reloaded.
Could you please check again and guide us? Also could you please share how can we find which script or content could be the issue for this?
URL

Thanks

**Reply by vovayatsyuk** [reply]
It's caused by the following code in `WilliamsCommerce_BreezeMirasvitSearchUltimate/js/search-auto-complete/autocomplete.js`:

```js
$(document).on("click", function (event) {
    event.stopPropagation(); // <-- this is the causer
    this.clickHandler(event);
}.bind(this));
```

Just remove this line.

**Reply by sandipansaha** [reply]
Hi Yatsyuk,

Thank you for your help. I have removed it and I guess it is working now.
May I ask how did you find that out? is there any technique you are using for?

Thanks

**Reply by vovayatsyuk** [reply]
You are welcome! Great site, by the way!

Nothing special, I just used search by code in chrome dev tools (I've assumed that some js may use `.stopPropagation` or `.stopImmediatePropagation` function that prevents click event from bubbling to the turbo listener).
